packages/node_modules/samples/browser-single-party-call/app.js

```diff
@@ -227,3 +233,37 @@ document.getElementById('dialer').addEventListener('submit', (event) => {
       // Implement error handling here
     });
 });
+
+window.addEventListener('load', () => {
+  const audio = document.getElementById('constraints-audio');
+  const video = document.getElementById('constraints-video');
+
+  if (navigator && navigator.mediaDevices && navigator.mediaDevices.enumerateDevices) {
```

Comment:
This would be a nice helper function to move into the main `meetings/index.js`. Not necessary for this ticket though. @taymoork2 don't you think?

Reply:
exposing `getSupportedDevice` (which should be renamed to `getSupportedDevices`) should be enough

Reply:
This is fine because it was to needed be run before the SDK is downloaded

Reply:
@taymoork2 Should I rename it now? Wondering if there is people already using this method that would be a breaking change?

Reply:
Probably the one you're exposing since its the first time it's being exposed as a public api
Internal getSupportedDevice can get renamed later if need be

Reply:
done!